CMakeLists.txt

```diff
@@ -257,7 +257,8 @@ endif()
 ########################################################################
 ## graphical interface
 ########################################################################
-cmake_dependent_option(BUILD_GUI "Build limeGUI application" ON "BUILD_CORE" OFF)
+option(GUI_TOGGLE "Manual switch to include/remove GUI components into/from project build" ON)
```

Comment:
Why is this needed? GUI inclusion in the project is already controlled by  "cmake -DBUILD_GUI=off" option